astro/src/content/docs/release-notes/index.mdx

```diff
@@ -339,6 +339,42 @@ Update dependencies.
 * Applications may now be designated as third-party applications. In addition to the prompting for authorization that comes with the `Custom OAuth Scopes` feature (see above), limitations are being added to how third-party applications may interact with FusionAuth.
   * Resolves [GitHub Issue #2723](https://github.com/FusionAuth/fusionauth-issues/issues/2723)
 * Applications can now be configured to prompt users to grant consent to requested OAuth scopes using the `/oauth2/consent` [themed](/docs/customize/look-and-feel/) page. See the OAuth [Scopes](/docs/lifecycle/authenticate-users/oauth/scopes) for more detail.
+
+  If you are using a custom theme, you will want to upgrade your theme to include this new page. Until you complete this upgrade, the default theme will be used for this new page. In the FusionAuth admin UI, the theme page will be named *OAuth consent prompt*.
+
+  The new consent prompt themed page requires a macro to be defined in the *Helpers* template in order to render scope consent form fields. You can find the new macro in the updated default theme or copy the macro below.
+  ```html
+  [#macro scopeConsentField application scope type]
```

Comment:
It's kind of a lot to put the whole thing here, but I'm not sure there's another place we could put it in docs and reference from here.

Reply:
Yeah, this is cumbersome. But so is having to add a single macro to a helpers file that could blow up your theme if you did something wrong and corrupted that file. Maybe we could do something like:

The new consent prompt themed page requires a macro to be defined in the *Helpers* template in order to render scope consent form fields. To get this new macro, you have two options:
* ** If you haven't edited the `Helpers` file in your theme **, you can copy the contents of the `Helpers` file from the default theme into your custom theme.
* If you have edited the `Helpers` file in your theme, you can copy the `scopeConsentField` macro from the `Helpers` file in the default theme and paste it at the bottom of the `Helpers` file in your custom theme.

Although I'm a little afraid people might just blindly follow those two bullets without reading the conditions.

Reply:
Do we need users to pick up 
```
[#function resolveScopeMessaging messageType application scopeName default]
```
as well?

Reply:
I tried to follow some of the other callouts for theme changes on this one. I think there is a page on upgrading the theme that has more advice on how to upgrade and test themes generally.

Reply:
Good catch. I was rushing to get this posted and missed that.

I updated with the function as well.